test/validators.js

```diff
@@ -4698,12 +4704,54 @@ describe('Validators', () => {
       valid: ['abc', 'de', 'a', ''],
       invalid: ['abcd'],
     });
+    test({
+      validator: 'isLength',
+      args: [{ max: 6, exact: 5 }],
+      valid: ['abcde', 'fffde', 'allkk'],
+      invalid: ['abcd', 'abcdef', 'vfd', 'ff', '', 'k'],
+    });
+    test({
+      validator: 'isLength',
+      args: [{ max: 6, exact: { } }],
+      valid: [],
+      invalid: ['abcd', 'abcdef', 'vfd', 'ff', '', 'k'],
+    });
+    test({
+      validator: 'isLength',
+      args: [{ min: 2, max: 6, exact: 5 }],
+      valid: ['abcde', 'fffde', 'allkk'],
+      invalid: ['bsa', 'vfvd', 'ff', '', 'k'],
+    });
+    test({
+      validator: 'isLength',
+      args: [{ exact: 2 }],
+      valid: ['fg', 'ff', 'po'],
+      invalid: ['bsa', 'vfvd', '', 'k'],
+    });
     test({
       validator: 'isLength',
       args: [{ max: 0 }],
       valid: [''],
       invalid: ['a', 'ab'],
     });
+    test({
+      validator: 'isLength',
+      args: [{ min: 5, max: 10, exact: [6, 8, 9] }],
+      valid: ['helloguy', 'shopping', 'validator', 'length'],
+      invalid: ['abcde', 'abcdefg'],
+    });
+    test({
+      validator: 'isLength',
+      args: [{ min: 5, max: 10, exact: { first: 6, second: 8, third: 9 } }],
```

Comment:
What would happen if one of the `exact` numbers is lower than `min` or higher than `max`? Should we add tests for that behaviour as well?

Reply:
Since the `exact` condition only runs when the Length of the String falls inside `[min, max]` range, if at all `exact` integer or one of the `exact` numbers is out of range `[min, max]`, the equality check for that case will return `false` and `isValid` won't be set to `true`.
Also, I have included the test condition for that behavior in the already present test with `args: [{ min: 5, max: 10, exact: [2, 6, 8, 9] }]`.